netlify/functions/robinhood-bot.cjs

```
@@ -0,0 +1,345 @@
+        const symbol = event.queryStringParameters?.symbol;
+        if (!symbol) {
+          throw new Error('Symbol parameter required');
+        }
+        data = await getQuote(symbol.toUpperCase());
+        break;
+
+      case 'order':
+        if (event.httpMethod !== 'POST') {
+          throw new Error('POST method required for orders');
+        }
+        const { symbol: orderSymbol, side, quantity, dryRun = true } = body;
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)